api/base.js

```diff
@@ -134,7 +134,7 @@ class Api {
       return cb(new Error('ERR_API_READY'))
     }
 
-    const action = msg.action
+    const action = _.isObject(msg) && _.isString(msg.action) ? msg.action : null
```

Comment:
why don't we just do just `const { action } = msg` ?

Reply:
ok simplified destruct